
Max Length for sentence = 44
Processing Epoch 00:   0%|                                                                                                                                                            | 0/2138 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "c:\Shaurya\Longformer2\LongformerCoLA\train.py", line 156, in <module>
  File "c:\Shaurya\Longformer2\LongformerCoLA\train.py", line 122, in train_model
    loss = loss_fn(classification_output, label)
  File "c:\Shaurya\Longformer2\LongformerCoLA\model.py", line 217, in encode
    return self.encoder(src_pos_encoded, src_mask, src_mask)
  File "C:\Users\Shaurya\AppData\Local\Programs\Python\Python39\lib\site-packages\torch\nn\modules\module.py", line 1511, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "C:\Users\Shaurya\AppData\Local\Programs\Python\Python39\lib\site-packages\torch\nn\modules\module.py", line 1520, in _call_impl
    return forward_call(*args, **kwargs)
  File "c:\Shaurya\Longformer2\LongformerCoLA\model.py", line 193, in forward
    x = layer(x, v, mask)
  File "C:\Users\Shaurya\AppData\Local\Programs\Python\Python39\lib\site-packages\torch\nn\modules\module.py", line 1511, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "C:\Users\Shaurya\AppData\Local\Programs\Python\Python39\lib\site-packages\torch\nn\modules\module.py", line 1520, in _call_impl
    return forward_call(*args, **kwargs)
TypeError: forward() takes 3 positional arguments but 4 were given